app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def recent_5_comments
+    comment.order('created_at Desc').limit(5)
```

Comment:
- [ ] For this to work, I would recommend you add the `s` to comment here like so:

  ```
  comments.order('created_at Desc').limit(5)
  ```